sdl-core/src/main/scala/io/smartdatalake/workflow/action/DataFrameActionImpl.scala

```diff
@@ -553,8 +553,4 @@ abstract class DataFrameActionImpl extends ActionSubFeedsImpl[DataFrameSubFeed]
         }
     }
   }
-
-  override def postExecFailed(implicit context: ActionPipelineContext): Unit = {
-    super.postExecFailed
```

Comment:
thanks for removing this, now there is less risk that someone overrides it and then wonders why the metrics are not executed.